src/physics.cpp

```diff
@@ -1137,7 +1137,7 @@ void inelastic_scatter(const Nuclide& nuc, const Reaction& rx, Particle& p)
 void sample_secondary_photons(Particle& p, int i_nuclide)
 {
   // Sample the number of photons produced
-  double y_t = p.wgt_ * p.neutron_xs_[i_nuclide].photon_prod /
+  double y_t = /*p.wgt_ */ p.neutron_xs_[i_nuclide].photon_prod /
```

Comment:
```suggestion
  double y_t = p.neutron_xs_[i_nuclide].photon_prod /
```

Reply:
Thanks for the suggestion. The code has been updated.